### ISSUE TITLE
Custom home page messed up some of the CSS on the Index page

### ISSUE THREAD

**Issue by Lastofthefirst** [eddiesigner/liebling#350]
I made a custom home page and moved the normal home page to /news. With the only changed file in the theme being the added home page, the /news page(old home page) now has broken css, the slider being the noticeable part. I poked around a bit but I can't gather why.

The site I am editing is live with these errors at https://serbc.ridvan.org.

I have also been having difficulty getting the normal post css to apply to the new home page's body content. I'm not sure if this issue is theme specific, or if its an error with my handlebars syntax.

**Comment by eddiesigner** [comment]
I really like what you did with the theme 😄, specially the intro section and the hero section.

I believe you already solved the issues? Both pages look good to me, the only thing I found is that you're trying to load the file `js/home.js` in the home page with this code:

```html
<script defer src="/assets/ js/home.js?v=21b5c5fbfe"></script>
```

That url is wrong because there is an empty space in between, the correct url should be:

```html
<script defer src="/assets/js/home.js?v=21b5c5fbfe"></script>
```

**Comment by Lastofthefirst** [comment]
Yes, thanks for your reply! Sorry I meant to get back here and apparently did not.

Figured it out with help from someone over at the [ghost forums](https://forum.ghost.org/t/custom-home-page-broke-liebling-theme-index-css/21482).

The issue was my formatter, which I had set to format on save, was adding those spaces. Since I'm new to handle bars those spaces were there by my first edit, so I thought that was just how it started. I thought "thats weird syntax but ok"

I'm glad you like it, if you have any feedback on the parrallaxish transition on mobile or desktop let me know. **Thanks for sharing the theme!**
